orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -11481,6 +11485,13 @@ definitions:
       icmp:
         $ref: '#/definitions/icmp_status'
     type: object
+  subscriberdb_sync_interval:
+    description: Interval in seconds between gateway requests to sync its subscriberdb
+      with the cloud.
+    example: 300
+    format: uint32
+    minimum: 60
```

Comment:
So I think it is good to include this validation here. However, I think we also need to include the validation when creating the proto -- because operators could set the "default" value in lte.yml to be less than 1m, which would cause weird shit to happen

So keep this northbound validation, but also add something in your "get sync interval" method that returns `max{1m, calculatedSyncInterval}`

Reply:
I added the extra check in `getSyncInterval()`. I stored the 60 second minimum in `subscriberdb/const.go`.